CONTRIBUTING.md

```diff
@@ -120,14 +122,237 @@ rework your branch. You should rework any relevant information from the GitHub
 comment section into your patch set because your patches are documented in the
 Git log, not the comment section.
 
-For more information on GitHub "force push" workflows see "[Why and how to
-correctly amend GitHub pull
-requests](http://blog.adamspiers.org/2015/03/24/why-and-how-to-correctly-amend-github-pull-requests/)".
-
 Your PR can contain more than one patch. Use as many patches as necessary to
 implement the request. Each PR should only cover one topic. If you mix up
 different items in your patches or PR, they will likely need to be reworked.
 
+#### Example first PR
+
+This section will guide you through how to create your first PR. We will
+create a branch and in that branch we will change *this document*. This
+section assumes no previous `git(1)` setup or knowledge:
+
+- Basic git setup:
+
+  Install the `git` package for your distribution. Next, if you haven't
+  already done this, you need to tell `git` your name and email address so these
+  can be recorded in your commits.
+
+  ```sh
+  $ git config --global user.email "you@example.com"
+  $ git config --global user.name "Your Name"
+  ```
+
+  > **Note:** The email address you specify here must match your primary GitHub email address.
+
+- Create your own copy of the repository you want to change by clicking "fork"
+  on https://github.com/kata-containers/community.
+
+  This will create your copy under `https://github.com/${your-github-username}` *on the github server*.
+
+- [Clone your fork](https://help.github.com/articles/cloning-a-repository):
+
+  "Cloning" is a term that means to create a *local copy* of a git repository
+  that generally lives on a remote server.
+
+  ```sh
+  $ user="your-github-username"
+  $ dir="$GOPATH/src/github.com/kata-containers"
+  $ mkdir -p "$dir"
+  $ cd "$dir"
+  $ git clone https://github.com/${user}/community
```

Comment:
and maybe if you are using `GOPATH`, you could use `go get` - sure, I understand the quandry here, and a preference to probably stick with vanilla `git`